data/presets/barrier/stile.json

```diff
@@ -11,5 +11,10 @@
     "tags": {
         "barrier": "stile"
     },
+    "removeTags": {
+        "barrier": "stile",
+        "stile": "*",
+        "material": "*"
```

Comment:
in my testing `material` was for some reason not removed

no idea why

Reply:
the `material` tag is preserved when the new preset (e.g. `barrier=gate`) also has the `material` field. This is as intended. :)